docs/content/en/_index.md

```diff
@@ -12,44 +12,25 @@ description: "K8ssandra documentation: architecture, configuration, guided tasks
 type: docs
 ---
 
-The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. Check out the sections at left based on your needs and be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
+The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. 
 
-{{% alert title="Tip" color="primary" %}}
-If you're impatient, jump right in with our **[install]({{< relref "install" >}})** topics!
-{{% /alert %}}
+Be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
 
-## What is K8ssandra?
+## Features for single- and multi-cluster Kubernetes environments
 
-K8ssandra is a cloud native distribution of Apache Cassandra® (Cassandra) meant to run on Kubernetes. Accompanying Cassandra is a suite of tools to ease and automate operational tasks. This includes metrics, data anti-entropy services, and backup tooling. As part of K8ssandra's installation process all of these components are installed and wired together freeing your teams from having to perform the tedious plumbing of components.
+| K8ssandra v1.4.x      | K8ssandra Operator v1.0.0 |
+| ----------- | ----------- |
+| K8ssandra v1.4.x is our initial implementation. It provides a set of separate Helm charts you can use to configure and deploy Apache Cassandra&reg; into a single-cluster, single-region Kubernetes environment. | K8ssandra Operator v1.0.0 is our most recent offering. It combines API, management, &amp; observability features under the control of a unified operator. Important enhancements include **multi-cluster** and **multi-region** support for Cassandra deployments in Kubernetes, which enables greater scalability and availability. Single cluster/region deployments are also supported with K8ssandra Operator.|
```

Comment:
Overall this is a good intro, but I'm unsure about this part of the description: `It combines API, management, &amp; observability features`

What were you meaning with the API portion?

We might just want to be more generic here - observability is fairly minimal in the operator deployment, purposefully so.

Reply:
By "API," I was thinking of Stargate.  I should probably specify the features by name: Stargate (API), Reaper (anti-entropy repairs), Medusa (backup/restore).

Reply:
Updated in most recent commit. Note that right after this merge (if approved), I will move the top-level doc home _index.md over to the common `docs` branch.  Right, it's not for the docs-v2 branch.